{
  try {
    return computation.compute();
  }
 catch (  Exception e) {
    CompilerServerApplication.LOGGER.error(e);
    return null;
  }
}
